src/docs/sdk/features.mdx

```diff
@@ -154,3 +154,17 @@ This is mostly useful on mobile clients where connectivity is often not availabl
 ## HTTP Proxy
 
 Ability to use an HTTP proxy. Often easy to implement using the existing HTTP client.  This should be picked up from the system config if possible or explicit config in the client options.
+
+## HTTP Client Integrations
+
+Add a breadcrumb for each outgoing HTTP request after the request finishes:
+
+- type: `http`
+- category: `http`
+- data: `url`, `method` (uppercase HTTP method), optionally `requestBodySize` and `responseBodySize` (in bytes)
+
+If Performance Monitoring is both supported by the SDK and enabled in the client application when the transaction is active a new `Span` must be created around the HTTP request.
+
+- HTTP request must be enhanced with a [`sentry-trace` HTTP header](/sdk/performance/#header-sentry-trace)
+- span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
+- when network error occurs, span status must be set to `INTERNAL_SERVER_ERROR`
```

Comment:
What is meant by "network error" here? There are many things that can happen like failed to resolve DNS, timeout, etc.

Note that `INTERNAL_SERVER_ERROR` is not a valid status (`internal_error` is).

Lacking more context, I guess `unknown_error` is more appropriate as a generic catch-all status, but I look forward to us simplifying the statuses in the near future, the current list is confusing and not well fitting the product.

Reply:
@rhcarvalho about the `unknown_error `, See -> https://github.com/getsentry/develop/issues/307#issuecomment-822240763
we've changed it because the description of `unknown_error` is actually:

> An unknown error raised by APIs that don't return enough error information

and in our case, network error is eg you've no connection, something happened that you could not really talk with the endpoint, that's why internal rather unknown.

Reply:
I agree that status are a bit confusing, but that's what we've right now, we can revisit this in the future

Reply:
I see. Reusing `internal_error` unfortunately confuses with a 500 response.

Reply:
indeed, but `unknown_error` is also 500 anyway, so it's not any better IMO

Reply:
To be sure we understand the same, while the status values (first column, `States`) is sort of fixed ([known by Relay](https://github.com/getsentry/relay/blob/55127c75d4eeebf787848a05a12150ee5c59acd9/relay-common/src/constants.rs#L283-L306)), the `Description` and `HTTP status code equivalent` are under our control; it is whatever we want to document and it is relatively easy to change. AFAIK nothing in the product depends on those descriptions.

Reply:
yep, but right now that's how its implemented in some of the SDKs (using inetrnal_error) and its the one that fits the most in the description https://develop.sentry.dev/sdk/event-payloads/transaction/
so unless we fix the SDKs and the status descriptions, let's keep it like this, I'll merge as it is.